### ISSUE TITLE
More context on the service identity preview

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#1507]
## Description
As a client creating a new service, I need to understand what the preview is telling me so that I can use the preview as intended. 

WHY are we building?
We prototyped a solution in Jan 2024 where an interactive preview helped to explain what a service was. But, the preview lacked some context and participants misunderstood what it was trying to tell them.

WHAT are we building?
An iteration of the service identity preview

VALUE created by our solution
We're not relying on the service identity preview to do all the work to explain what a 'service' is. But, our clients do not misunderstand what the preview is explaining to them.

## Documentation and Artifacts
[Brainstorming Miro](https://miro.com/app/board/uXjVNR1bpWk=/?moveToWidget=3458764579684427886&cot=14)
[Research presentation for more details](https://docs.google.com/presentation/d/1rfh8Rr-PWqvzEbEbPhBSbZRXSBjufN-_WfFWh6iCorw/edit#slide=id.p1)

## Acceptance Criteria
When a new client who hasn't attended a demo is creating a new service, they can doublecheck their understanding of a service and the required fields in the 'service creation' flow using the preview without misunderstanding.

- [ ] Cypress UI tests if needed.
- [ ] Generate appropriate log messages so that executions of this feature can be tracked
- [ ] Can misuse of this feature cause harm? If yes, create an alert
- [ ] Update the status of related findings, insights, and hypotheses on the Research Airtable
- [ ] Once change/fix/feature is implemented, link relevant Airtable records to design artifacts (Figma)

* A11y
* Bilingualism
* Privacy considerations
* Security controls in place
* Measuring success and metrics

## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
- [The service identity preview needs more context. When we drew attention to the preview during the sessions, it was clear the text bubble was representing a chat function but it wasn’t clear it was a text message (e.g. it could be a message on a Teams chat).](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwtsq6Gy7TDVGL36/rec7nUjfpSB0Zsvvf?copyLinkToCellOrRecordOrigin=gridView&blocks=hide)

## QA Steps
- [ ] Tested in a realistic production scenario

**Comment by jzbahrai** [comment]
@amazingphilippe we did fable testing around this? I remember a service preview page we were testing? Is this card still relevant?
